orchestrator/gorc/src/commands/keys/cosmos/add.rs

```diff
@@ -0,0 +1,14 @@
+use abscissa_core::{Command, Options, Runnable};
+
+#[derive(Command, Debug, Default, Options)]
+pub struct AddCosmosKeyCmd {
+    #[options(short = "n", long = "name", help = "add private key [name]")]
```

Comment:
I used free for arguments instead of flags. What do you think?

Reply:
like args over flags personally